jmsSource/README.md

```diff
@@ -241,9 +241,13 @@ INSERT JMSMessageType(myObj)
 ### Select Statements <a name="select" id="select"></a>
 
 In order to read messages from a queue, (**NOT** a queueListener), a VAIL SELECT statement must be used. The SELECT statement 
-must have two query parameters: `operation` and `queue`. Currently there is only one `operation` that is supported, which is 
-the "read" `operation`. In the future, there may be different SELECT Operations. The `queue` parameter is the name of the 
-queue from which to read. The following is an example of a Procedure created in VANTIQ Modelo querying against a JMS Source:
+must have two query parameters: `operation` and `queue`. A third optional parameter, `timeout`, can be specified as well. 
+Currently there is only one `operation` that is supported, which is the "read" `operation`. In the future, there may be 
+different SELECT Operations. The `queue` parameter is the name of the queue from which to read. The `timeout` parameter can be 
+used to force the queue message consumer to wait up to `timeout` milliseconds for a queue message to arrive. The `timeout` 
+parameter must be a **non-negative integer**, otherwise it will be ignored. If the `timeout` value is set to 0, this will make 
```

Comment:
We might want to consider treating `0` the same as missing.  That is, require a positive integer and don't allow indefinite wait.  Really indefinite won't work anyway -- the user can get around it by waiting for an hour (which will fail when the procedure dies).

Reply:
Sounds good, I'll change this to not allow 0.